app/classifier/tasks/survey/choice.cjsx

```diff
@@ -52,7 +52,7 @@ module.exports = React.createClass
     onCancel: Function.prototype
 
   getInitialState: ->
-    answers: {}
+    answers: @props.annotation?.answers || {}
```

Comment:
There isn't. I didn't remove this line and replaceit with `annotation : { answers: {} }` because I didn't want to change all the occurences of `@@state.answers`. I can do that, if you prefer though.